src/renderer-process/js/modulation-view.js

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2019 Colin Clark
+
+Licensed under the MIT license.
+https://github.com/colinbdclark/bubbles/raw/master/LICENSE
+*/
+
+"use strict";
+
+fluid.defaults("bubbles.modulationView", {
+    gradeNames: "fluid.viewComponent",
+
+    model: {
+        value: 1.0
+    },
+
+    modelRelay: [
+        {
+            source: {
+                context: "modulationMatrixView",
+                segs: ["{that}.options.modulationName"]
+            },
+
+            target: "{that}.model.value",
+
+            singleTransform: {
+                type: "fluid.transforms.identity"
+            }
+        }
+    ],
+
+    bindings: {
+        "value": {
+            selector: "textField",
+            path: "value",
+            rules: {
+                domToModel: {
+                    "": {
+                        transform: {
+                            type: "fluid.transforms.stringToNumber",
+                            inputPath: ""
+                        }
+                    }
+                },
+                modelToDom: {
+                    "": {
+                        transform: {
+                            type: "fluid.transforms.numberToString",
+                            inputPath: ""
+                        }
+                    }
+                }
+            }
+        }
+    },
+
+    events: {
+        afterRendered: null
+    },
+
+    listeners: {
+        "onCreate.render": {
+            funcName: "bubbles.modulationMatrixView.render",
+            args: "{that}"
+        },
+
+        "afterRendered.applyBinding": {
+            funcName: "gpii.binder.applyBinding",
+            args: "{that}"
+        }
+    },
+
+    strings: {
+        id: "@expand:fluid.allocateGuid()",
```

Comment:
You could just use your own id?

Reply:
As in, the component's ID supplied by Infusion? Or something else?

Reply:
Yes, simply {that}.id